ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -95,3 +95,5 @@ sunbird_collectionEditorURL=collection-editor/index.html
 sunbird_contentEditorURL=content-editor/index.html
 sunbird_genericEditorURL=generic-editor/index.html
 sunbird_keycloak_public_key={{core_vault_sunbird_sso_publickey}}
+sunbird_bot_configured=true
```

Comment:
is this going to be mandatory from 3.0 ? i mean for adopters ?

Reply:
if this config is not given or given as false then the bot will be disabled

Reply:
Then this need to be variablized @rajeevsathish ,